<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crittografia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="userMessageBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cryptButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Crittografa</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="convertedLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveFile">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>111</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Salva su file</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="invertCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Inverti crittografia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="swapButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Swap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileReadButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>141</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Lettura da file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
